ludwig/encoders/text_encoders.py

```diff
@@ -297,19 +286,17 @@ def output_shape(self) -> torch.Size:
     def input_dtype(self):
         return torch.int32
 
+    @classmethod
+    def get_fixed_preprocessing_params(cls, encoder_params: Optional[Dict]) -> Dict[str, Union[None, int]]:
+        return {
+            "tokenizer": "hf_tokenizer",
+            "pretrained_model_name_or_path": "google/mt5-base",
```

Comment:
It looks like we dropped the `feature.pretrain_model_name_or_path` part here. I'm not sure how that was being used before. Can you explain why it was okay to drop that part?

Reply:
Ah, I see, it's needed for `build_sequence_matrix`. So this is actually a place where your improved interface will be very useful.